site/docs/try/configuration.md

```diff
@@ -15,20 +15,11 @@ docker run -p 8080:8080 projectnessie/nessie \
 ## Core Nessie Configuration Settings
 
 ```properties
-# which type of version store to use: JGIT, INMEMORY, DYNAMO. JGIT is for local testing, DYNAMO preferred for production
-nessie.version.store.type=DYNAMO
+# which type of version store to use: ROCKSDB, INMEMORY.
+nessie.version.store.type=ROCKSDB
 
-# path if using JGIT
-nessie.version.store.jgit.directory=/tmp/jgit
-
-## Dynamo version store specific configuration
-# should Nessie create its own dynamo tables
-nessie.version.store.dynamo.initialize=false
-
-## Dynamo Configuration
-quarkus.dynamodb.aws.region=us-west-2
-quarkus.dynamodb.aws.credentials.type=DEFAULT
-# quarkus.dynamodb.endpoint-override=http://localhost:8000
+# path if using RocksDB
+nessie.version.store.rocks.directory=/tmp/rocks-nessie
```

Comment:
Isn't this effectively the same as "in memory"? Should we suggest a non-transient location perhaps?

Reply:
`/tmp/` has the advantage of being accessible to all users (unlike `/var/lib` or so).

Note: `/tmp` is not always transient in Linux though. [FHS 3.0](https://refspecs.linuxbase.org/FHS_3.0/fhs/ch03s18.html) states that you should not _assume_ that files survive reboots, but at least Ubuntu does not clear `/tmp` nor does it periodically do that (anymore). I suspect, clearing `/tmp`just caused too many problems (a lot of things place important, long-living things there - e.g. Java's hsperfdata, systemd, X11, etc).

Reply:
fair enough.